app/views/projects/_form.html.erb

```diff
@@ -62,6 +61,8 @@
     <%= render :partial=> "projects/select_add_current_user_to_project" %>
 <% end %>
 
+<%= render partial: "assets/controlled_vocab_annotations_form_property", locals: { resource: @project, property: :disciplines} -%>
```

Comment:
I think it would be better if this was placed next to Topics, since the are related, or better replace both with an updated  `assets/controlled_vocab_annotations_form_properties`